Messenger/common.json

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Can you remove this file. This was updated recently to reflect the latest interface for Messenger.

Reply:
@npoltorapavlo - Can you check if it's ok to add Common.json file here in Messenger.

Reply:
I can't say any of this is "ok" as my understanding is interfaces should be in ThunderInterfaces and not in rdkservices.

Reply:
@npoltorapavlo, that is not how rdkservices works at present and I'm not trying to get into a religious war here. The fact is that you added a reference to a file that did not exist and that broken link broke the documentation build. We either have to restore the file the way it was, or we have to add the referenced file (common.json) somewhere in our repository.